devops/Makefile

```diff
@@ -39,6 +39,20 @@ export FORMIO_ROOT_EMAIL := $(or ${FORMIO_ROOT_EMAIL}, dev_sabc@gov.bc.ca)
 export MONGODB_URI := $(or ${MONGODB_URI}, $$MONGODB_URI)
 export QUEUE_PREFIX := $(or $(QUEUE_PREFIX), {sims-local})
 export QUEUE_DASHBOARD_TOKEN_EXPIRATION_SECONDS := $(or $(QUEUE_DASHBOARD_TOKEN_EXPIRATION_SECONDS), 3600)
+export VOLUME_CAPACITY := $(or $(VOLUME_CAPACITY), 2Gi)
+export SC_MONGO := $(or ${SC_MONGO}, netapp-block-standard)
```

Comment:
Is there a reason to have this as a configuration instead of a hard-coded value in the yaml?

Reply:
no it was in the param file as a parameter, so just refactored it. I can make it a hard-coaded value.